bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/image/IconType.java

```diff
@@ -1,37 +1,64 @@
-package de.agilecoders.wicket.core.markup.html.bootstrap.image;
-
-import de.agilecoders.wicket.core.markup.html.bootstrap.behavior.CssClassNameAppender;
-import de.agilecoders.wicket.core.markup.html.bootstrap.behavior.ICssClassNameModifier;
-import de.agilecoders.wicket.core.markup.html.bootstrap.behavior.ICssClassNameProvider;
-
-import org.apache.wicket.util.lang.Args;
-
-/**
- * References all available icons inside the icon sprite.
- */
-public abstract class IconType implements ICssClassNameProvider, ICssClassNameModifier {
-    /** CSS classname. */
-    private String cssClassName;
-
-    /**
-     * Constructor.
-     *
-     * @param cssClassName The css class name of the icon reference
-     */
-    protected IconType(String cssClassName) {
-        Args.notEmpty(cssClassName, "cssClassName");
-        this.cssClassName = cssClassName.toLowerCase();
-    }
-
-	/**
-	 * @return the cssClassName
-	 */
-	protected String getCssClassName() {
-		return cssClassName;
-	}
-    
-    @Override
-    public CssClassNameAppender newCssClassNameModifier() {
-        return new CssClassNameAppender(getCssClassName());
-    }
-}
+package de.agilecoders.wicket.core.markup.html.bootstrap.image;

```

Comment:
Now I see why I wasn't able to comment on your other PR - because I needed to check it with "No whitespace" option enabled.
Such full rewrite of the file due to different whitespace should be avoided.

Reply:
I don't see why all the whitespaces were changed. The actual code is using spaces and so am I in eclipse.
I have not this problem with diff in tortoisegit but I do see it in eclipse. What did I miss?

Reply:
It seems to have something something to do with linefeed and not space/tab.
https://bugs.eclipse.org/bugs/show_bug.cgi?id=342372#c154

Reply:
Yes, the problem is that your Eclipse have changed the newlines to Windows style.